robot/control/Inc/motion-control/RobotEstimator.hpp

```diff
@@ -43,21 +68,68 @@ class RobotEstimator {
     void getState(Eigen::Matrix<float, numStates, 1>& state);
 
 private:
-    static constexpr float processNoise = 0.05;
-    static constexpr float encoderNoise = 0.04;
-    static constexpr float gyroNoise = 0.005;
-    static constexpr float initCovariance = 0.1;
 
+    static constexpr float processNoise = 0.05;  /**< Random noise caused by external influences on the robot */
```

Comment:
I would say processNoise represents any unmodeled influence on the states. It could be assumptions about the constant velocity built into the transition matrices, modeling errors on the actual robot properties like weight, it could be completely unmodeled things like wheel slip.